README.adoc

```diff
@@ -20,6 +20,22 @@ image:https://badges.gitter.im/eclipse/microprofile-reactive.svg[link="https://g
 
 = MicroProfile Reactive Streams Operators
 
+'''
+== How to contribute
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
```

Comment:
Should it mention:

* Sign the Eclipse CLA
* Sign-off all your commit

Reply:
Good point, let me add that quickly

Reply:
Done